tests/functional/spec/features/new_calculated_summary.spec.js

```diff
@@ -0,0 +1,331 @@
+import FirstNumberBlockPage from "../../generated_pages/calculated_summary/first-number-block.page.js";
+import SecondNumberBlockPage from "../../generated_pages/calculated_summary/second-number-block.page.js";
+import ThirdNumberBlockPage from "../../generated_pages/calculated_summary/third-number-block.page.js";
+import ThirdAndAHalfNumberBlockPage from "../../generated_pages/calculated_summary/third-and-a-half-number-block.page.js";
+import SkipFourthBlockPage from "../../generated_pages/calculated_summary/skip-fourth-block.page.js";
+import FourthNumberBlockPage from "../../generated_pages/calculated_summary/fourth-number-block.page.js";
+import FourthAndAHalfNumberBlockPage from "../../generated_pages/calculated_summary/fourth-and-a-half-number-block.page.js";
+import FifthNumberBlockPage from "../../generated_pages/calculated_summary/fifth-number-block.page.js";
+import SixthNumberBlockPage from "../../generated_pages/calculated_summary/sixth-number-block.page.js";
+import CurrencyTotalPlaybackPageWithFourth from "../../generated_pages/calculated_summary/currency-total-playback-with-fourth.page.js";
+import SetMinMaxBlockPage from "../../generated_pages/calculated_summary/set-min-max-block.page.js";
+import CurrencyTotalPlaybackPageSkippedFourth from "../../generated_pages/calculated_summary/currency-total-playback-skipped-fourth.page.js";
+import UnitTotalPlaybackPage from "../../generated_pages/calculated_summary/unit-total-playback.page.js";
+import PercentageTotalPlaybackPage from "../../generated_pages/calculated_summary/percentage-total-playback.page.js";
+import NumberTotalPlaybackPage from "../../generated_pages/calculated_summary/number-total-playback.page.js";
+import CalculatedSummaryTotalConfirmation from "../../generated_pages/calculated_summary/calculated-summary-total-confirmation.page.js";
+import SubmitPage from "../../generated_pages/calculated_summary/submit.page";
+import ThankYouPage from "../../base_pages/thank-you.page.js";
+
+describe("Feature: Calculated Summary", () => {
+  describe("Given I have a Calculated Summary", () => {
+    before("Get to Calculated Summary", () => {
+      browser.openQuestionnaire("test_new_calculated_summary.json");
```

Comment:
Make this a test case like the time-out modal ones to avoid the duplicate test?. This and the legacy format tests are identical apart from the schema loaded right?.

Reply:
Updated this 👍